hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterServiceImpl.java

```diff
@@ -933,11 +954,11 @@ private void shutdownNodes() {
         hazelcastInstance.getLifecycleService().shutdown();
     }
 
-    public void initialClusterState(ClusterState clusterState) {
+    public void initialClusterState(ClusterState clusterState, Version version) {
         if (node.joined()) {
```

Comment:
When is this invoked? Aha, probably on JoinFinalize, but if there's no join in a single-node cluster?